cmd/snap-update-ns/bootstrap.go

```diff
@@ -30,11 +30,11 @@ package main
 
 // The bootstrap function is called by the loader before passing
 // control to main. We are using `preinit_array` rather than
-// `init_array` because the Go linker adds its own initialisation
+// `init_array` because the Go linker adds its own initialization
```

Comment:
Actually the British spelling is "initialisation", so I'm not sure if we should change this.

Reply:
ah well my spellchecker isn't British so it didn't notice this :-) 

But I will remove this